savaslabs/siafg
# Issue #26993529: Generate static JSON files for airtable data.

This PR pulls into calls to airtable out of the app completely and instead generates static JSON files. Everything else on the [QA list](https://savaslabs.teamwork.com/#/tasklists/2180085) (except for footer) should also be working. Some of that is pushed to master already, some is on develop, and I probably snuck some of it on this one too 🙃 

The `data-scripts` code could be more efficient, but I was focused on just getting it done for time being.

## To Test
1. Move through app and ensure everything is working as expected — I think I found all the spots where data got weird, but another sets of eyes might reveal something I missed.
1. Run `yarn node data/data-scripts.js`. This should generate `json` files into `src/assets/data`. I committed already generated ones so the app has it.
